resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -179,7 +179,7 @@ s3.download.object.description=Download
 s3.download.object.progress=Downloading ''{0}''
 s3.load_more=load more...
 s3.name=Name
-s3.open.file_too_big=Cannot open files larger than {0} in the editor!
+s3.open.file_too_big=Cannot open files larger than {0}!
```

Comment:
Maybe more like: The editor cannot open files larger than x

Reply:
That would be as long as the previous one which would still get cut off

Reply:
If you set a title for `notifyError` does that solve the cut off issue?

Reply:
Drop the '!'

Future idea..offer to download locally if https://plugins.jetbrains.com/plugin/11363-large-file-editor is installed

Never used it so no clue how it works

Reply:
Sure, and we can set content on notifyError instead which is fixes the overhang